test/Github/Tests/ResultPagerTest.php

```diff
@@ -97,9 +101,12 @@ public function testFetch()
         $result = 'foo';
         $method = 'bar';
         $parameters = array('baz');
-        $api = $this->getMock('Github\Api\ApiInterface');
+        $api = $this->createMock('Github\Api\ApiInterface');
```

Comment:
`createMock` was added in PHPUnit 5.4 and PHPUnit 5.X is min PHP 5.5

https://github.com/sebastianbergmann/phpunit/wiki/Release-Announcement-for-PHPUnit-5.4.0#new-features